src/core/raster/qgsrasterlayerrenderer.cpp

```diff
@@ -282,6 +282,8 @@ QgsRasterLayerRenderer::QgsRasterLayerRenderer( QgsRasterLayer *layer, QgsRender
   }
 
   mClippingRegions = QgsMapClippingUtils::collectClippingRegionsForLayer( *renderContext(), layer );
+
+  mFeedback->setRendererUsage( rendererContext.rendererUsage() );
```

Comment:
This is ugly. We may as well just make a "setRenderContext"/"renderContext" member in QgsRasterBlockFeedback rather then add pieces of render context bit by bit.